multimod/internal/prerelease/prerelease.go

```diff
@@ -23,7 +23,6 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/go-git/go-git/v5"
```

Comment:
This shouldn't have been removed.

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-go-build-tools/pull/210/commits/8606050c7b23c2f1a9a44c4910751e40856698d9